chain/jsonrpc/src/lib.rs

```diff
@@ -312,6 +314,9 @@ impl JsonRpcHandler {
         Ok(match request.method.as_ref() {
             // Handlers ordered alphabetically
             "block" => process_method_call(request, |params| self.block(params)).await,
+            "broadcast_tx" => {
```

Comment:
The word `broadcast` bothers me a bit.
It's not a broadcast, and it exposes an implementation detail.
In the documentation this method is described as "Send transaction".

That's bikeshedding, and I don't suggest to rename existing methods, but maybe this new method can have a better naming.

Reply:
I decided it's worth discussing
https://near.zulipchat.com/#narrow/stream/295302-general/topic/Method.20name.20for.20adding.20transaction.20to.20the.20chain/near/397287445